src/app.js

```diff
@@ -171,16 +171,17 @@ async function getVideoData () {
         error(`failed to parse the lesson page '${urlValue}'}`)
       }
     } else {
-      let lessonURLs = []
       success('The URL is a playlist or series')
 
       // get the urls of the lessons
-      const re = /<a class="flex h-100 absolute top-0 right-0 bottom-0 left-0" style="pointer-events:all;" href="(\/lessons\/.+?)"/g
```

Comment:
actually change was just `pointer-events:all;` -> `pointer-events:all`, but I think that it is to fragile to match all css classes and style

Reply:
Just make a quick fix 😁.

Reply:
yeah ok